mautic/mautic
# Category in one place

This PR removes all the category menu items from left menu and adds one Category menu item to the right menu.

### How to test
1. Apply this PR and clear cache.
2. Make sure menu items are like described above.
3. You can see all categories from all bundles at one place when you open the Categories form the right menu.
4. You can filter by Type (by bundles)
5. You can create new categories and assign them to bundles.
6. You can edit old categories and you cannot change their type (assign it to different bundles). That would cause chaos when the category has some items in it. So I rather don't allow to change it.
7. Try to do crazy things with categories like people do.
